src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Resources/SR.resx

```diff
@@ -129,6 +129,9 @@
   <data name="ExtractTo_CodeBehind_Title" xml:space="preserve">
     <value>Extract block to code behind</value>
   </data>
+  <data name="Generate_Method_Title" xml:space="preserve">
+    <value>Generate '{0}' Method</value>
```

Comment:
I think we should call this "Generate Event Handler" so its not confused with future support for Roslyn's Generate Method action, and we don't get users confused why we don't offer it anywhere else.

Reply:
Good point, I will double check with Leslie on this before making changes.